src/Apps/Sell/SellFlowContext.tsx

```diff
@@ -143,7 +143,7 @@ export const SellFlowContextProvider: React.FC<SellFlowContextProviderProps> = (
       logger.error("Error creating submission.", err)
       sendToast({
         variant: "error",
-        message: err?.[0]?.message || "Something went wrong.",
+        message: "Something went wrong." + ` ${err?.[0]?.message} || ""`,
```

Comment:
Displaying the text and an error message in case something goes wrong to not only display something like "Unexpected item at ...."